zerver/lib/generateData.py

```diff
@@ -0,0 +1,273 @@
+from __future__ import absolute_import
+from __future__ import print_function
+import itertools
+import json
+import random
+from typing import List, Dict, Any, Text, Optional
+from six.moves import range
+
+def loadConfig():
```

Comment:
Stylistically, we don't use camelCase in Zulip; we use underscores_for_words, e.g. `load_config`.  We should change everything in this file, including the filename, to match Zulip's style.